sdk/storage/azure-storage-file/apireview/azure/storage/file/aio/models.pyi

```diff
@@ -0,0 +1,34 @@
+# Stubs for azure.storage.file.aio.models (Python 3)
```

Comment:
I don't think any of this should be public, unless the attributes in the class itself matter.
If that's the case, typing "AsyncPagedItem" is not sufficient and these types should be used instead

Reply:
#7799